health_ping.py

```diff
@@ -198,9 +198,8 @@ def process_env(c_name, e_id, endpoint, endpoint_type, component, env_attributes
   env_data = {}
   update_sc = False
   app_version = None
-  if endpoint_type == 'info':
-    app_version = get_build_image_tag(output)
-  if app_version:
+  # HEAT-567 - get app version from build image tag on health or info
+  if app_version := get_build_image_tag(output):
```

Comment:
will this add duplicates to the streams? Had this with the old script. Where the build image tag appears in both the health and info endpoints.